_includes/configure-registry-json.md

```diff
@@ -23,7 +23,9 @@ The `registry.json` file must contain the following contents, where `myorg` is r
 }
 ```
 
-You can use the following methods to create a `registry.json` file based on the user's operating system.
+In Docker Hub, you can download the registry.json file for your organization or copy the specific commands to create the file for your organization. To download the file or copy the commands from [Docker Hub](http://hub.docker.com), go to **Organizations** > **_Your Organization_** > **Settings** > **Enforce Sign-in**.
```

Comment:
When the Admin selects the Settings tab, is the Enforce Sign-in located in the left navigation? Is it a button in the UI?

Reply:
In the left navigation.

Reply:
Suggestion: 
 In Docker Hub, you can download the registry.json file for your organization or copy the specific commands to create the file for your organization. To download the file or copy the commands from [Docker Hub](http://hub.docker.com), go to **Organizations** > **_Your Organization_** > **Settings** . 

Select Enforce Sign-in and continue with the on-screen instructions for Windows, Mac, or Linux.

Reply:
Thanks, @jerae-duffin. I separated it into ordered steps in the following commit. PTAL